cmd/habitat/community/consensus/raft/helpers.go

```diff
@@ -0,0 +1,53 @@
+package raft
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/eagraf/habitat/pkg/compass"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	Localhost  = "localhost"
+	DockerHost = "0.0.0.0"
+	Protocol   = "http"
+
+	MultiplexerPort = "6000"
+	RPCPort         = "6001"
+)
+
+func getMultiplexerAddress() string {
+	ip, err := compass.LocalIPv4()
+	if err != nil {
+		log.Fatal().Msgf("error getting local IP address: %s", err)
+	}
+
+	return fmt.Sprintf("%s:%s", ip.String(), MultiplexerPort)
+}
+
+func getRPCAddress() string {
+	host := Localhost
+	if compass.InDockerContainer() {
+		host = compass.Hostname()
+	}
+
+	return fmt.Sprintf("%s:%s", host, RPCPort)
+}
+
+func getServerID(communityID string) string {
+	nodeID := compass.NodeID()
+	return fmt.Sprintf("%s#%s", nodeID, communityID)
+}
+
+func getCommunityAddress(communityID string) string {
+	return fmt.Sprintf("%s/%s", getMultiplexerAddress(), communityID)
+}
+
+func getCommunityExternalAddress(communityID string) string {
```

Comment:
can you give some examples of what this looks like?

Like, I create a community, get address _. Someone else can reach me at _. My ServerID is _.

Reply:
yeah this ended up being a little confusing because of the path it took to get here. I ended up removing a couple of the helpers that we don't need anymore.

Communities are identified by a UUID, and get a URL for Raft stuff that looks like `http://172.26.0.4:6000/4e58ebe5-3048-4c4c-a697-7a9dabc75583`. The server id is the combination of the community uuid and the node uuid (e.g. `4e58ebe5-3048-4c4c-a697-7a9dabc75583#8f86daf7-7a28-4325-aa1b-45d0aebb70c7`